### DISCUSSION TITLE
Display PR "checks" from Forge

### DISCUSSION THREAD

**Discussion by andreasjansson** [magit/forge#508] [Ideas]
As far as I can tell, it's not possible to view the status of "checks" on Github pull requests. By "checks", I mean these:
<img width="980" alt="image" src="https://user-images.githubusercontent.com/713993/184009857-ec8fcc1f-09bb-4cb3-8b60-32e3dab8444c.png">

It would be fantastic to be able to see the status of these checks from within Forge.

**Comment by tarsius** [comment]
Also see #92.
